### DISCUSSION TITLE
Datenschutz.bat + WindowsSpyBlocker

### DISCUSSION THREAD

**Discussion by AndroidUserwithMiuiu** [McAlex777/Datenschutz#3] [Q&A]
Hallo Alex777,

soweit ich in Erfahrung bringen konnte verstehst du auch deutsch. Von daher tue ich mich leichter am späten Abend in meiner Muttersprache dir zu schreiben.

Und zwar hab ich dein Konzept soweit umgesetzt (mit Whitelisting der Windows-Firewall). Nur Frage ich mich, ob die zusätzlichen Blocking-Firewall-Rules von WindowsSpyBlocker (https://crazymax.dev/WindowsSpyBlocker/app/telemetry/firewall/) nicht noch mehr Sicherheit bringen? Da so ja trotzdem die IPs über die erlaubten Standard-Firewall-Rules von dir erreichbar sind (über svchost, usw..). Hast du hier schon Erfahrung gemacht mit dem zusätzlichen ausführen dieses Tools?

Vielen Dank für ein paar Infos.

Mfg

**Comment by 1kartafal4** [comment]
Muss man die Befehle "fwi export" und "fwi fw_whitelisting" (Firewwall-Script) für noch weniger Telemetrie durch Seitens Microsoft etc. zusätzlich ausführen?
Habe "fwi" und "fwi addjob" ausgfeführt" und funktioniert so.
WindowsSpyBlocker benutze ich auch. Zusätzlich zu den drei Rules mittels Konsolenfenster (extra, spy, updates) habe ich mittels dem HostsMan Programmes https://crazymax.dev/WindowsSpyBlocker/blocking-rules/hosts/
noch die sechs hier heruntergeladen https://github.com/crazy-max/WindowsSpyBlocker//tree/master/data/hosts
importiert, und zusammengefügt.
Die hosts Datei ist auf 110 kB angewachsen und die ausgehenden Windows FirewallRegeln auch. Sollte doch so gemeinsam funktionieren?

Gruß
